I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 53, in script
    show Elliot at mid
  File "game/script.rpy", line 53, in <module>
    show Elliot at mid
NameError: name 'mid' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 53, in script
    show Elliot at mid
  File "renpy/ast.py", line 1197, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "renpy/ast.py", line 1156, in show_imspec
    at_list = [ renpy.python.py_eval(i) for i in at_list ]
  File "renpy/python.py", line 2276, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "renpy/python.py", line 2269, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 53, in <module>
    show Elliot at mid
NameError: name 'mid' is not defined

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
The Hollows 1.0
Sun Jan 30 16:11:39 2022
